v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_globals.py

```diff
@@ -1686,7 +1686,6 @@ async def __cancel_setting_listener(self, setting_object: dict):
 #                           UTIL FUNCTIONS                           #
 ######################################################################
 
-
```

Comment:
Was this an automated formatting change?
Maybe we should set up a pre-commit hook to make sure we do not keep on changing white space for no good reason?